### DISCUSSION TITLE
How to record programs running in the background?

### DISCUSSION THREAD

**Discussion by sino1641** [ActivityWatch/activitywatch#707] [Q&A]
I am a common user of activitywatch who would like to know how to record specific programs durations even running in the background?
I have also searched the documentations(`docs.activitywatch.net`) but nothing related was found.
Looking forward to your help, much thanks!

**Comment by ErikBjare** [comment]
There currently no way to do this without writing some code, but one could write a custom watcher which tracks if a specific application/window is running/open, or keep track of all of them (much more data-intensive).

Could you provide an example of an application you would like to track in the background? (just so I understand the use-case)
